pkg/distributors/contracts/MerkleOrchard.sol

```diff
@@ -37,7 +37,7 @@ contract MerkleOrchard {
     // token > distributor > balance
     mapping(IERC20 => mapping(address => uint256)) public suppliedBalance;
 
-    event DistributionAdded(address indexed token, uint256 amount);
+    event DistributionAdded(address indexed token, bytes32 merkleRoot, uint256 amount);
     event DistributionPaid(address indexed user, address indexed token, uint256 amount);
```

Comment:
Shouldn't these have the channel id?

Reply:
in place of `token`?

Reply:
`DistributionPaid` can refer to multiple channels as it's currently emitted once for each token being claimed

Reply:
No, I meant as an extra argument. If there are multiple channels for `token` (e.g. multiple teams making distributions in the same token), `DistributionPaid` isn't enough to figure out which channel was claimed.

Reply:
I've split up where we emit the `DistributionPaid` (now `DistributionSent`) event to support this.